fineract-provider/src/main/java/org/apache/fineract/infrastructure/hooks/service/HookWritePlatformServiceJpaRepositoryImpl.java

```diff
@@ -282,15 +282,10 @@ private void validateHookRules(final HookTemplate template, final Set<HookConfig
             if (conf.getFieldName().equals(payloadURLName)) {
                 try {
                     final WebHookService service = processorHelper.createWebHookService(fieldValue);
-                    service.sendEmptyRequest();
-                } catch (RetrofitError re) {
-                    // Swallow error if it's because of method not supported or
-                    // if url throws 404 - required for integration test,
-                    // url generated on 1st POST request
-                    if (re.getResponse() == null) {
-                        String errorMessage = "url.invalid";
-                        baseDataValidator.reset().failWithCodeNoParameterAddedToErrorCode(errorMessage);
-                    }
+                    service.sendEmptyRequest().execute();
+                } catch (IOException re) {
+                    String errorMessage = "url.invalid";
```

Comment:
just wondering if this part is taken care ?  "  // Swallow error if it's because of method not supported or
                    // if url throws 404 - required for integration test,
                    // url generated on 1st POST request"

Reply:
I was also wondering, but the integration tests all still pass now, so looks like this actually wasn't needed.